src/Scaler.Tests/CosmosDbScalerServiceTests.cs

```diff
@@ -156,6 +150,94 @@ public async Task GetMetricSpec_ReturnsNormalizedMetricName()
             Assert.Equal(
                 "cosmosdb-partitioncount-example-com-dummy-lease-database-id-dummy-lease-container-id-dummy-processor-name",
                 response.MetricSpecs[0].MetricName);
+        } 
+
+        [Fact]
+        public async Task IsActive_ThrowsOnMissingConnectionAndConnectionFromEnv()
```

Comment:
We want to add test case for one of it missing?

Reply:
Thanks for pointing out, I was only testing in UTs if it passes with only "FromEnv" properties when connection was not given. Updated it to checking for if either of them is missing.